mst/postgresql-read-replica.md

```diff
@@ -0,0 +1,40 @@
+---
+title: Create a read-only replica of PostgreSQL
+excerpt: Learn how to create a read-only replica for PostgreSQL service on Managed Service for TimescaleDB 
+product: mst
+---
+
+# Create a read-only replica of PostgreSQL service
+
+PostgreSQL read-only replicas allow you to perform read-only queries against
+the replica and reduce the load on the primary server. You can optimize query
+response times across different geographical locations because the replica can
+be created in different regions or on different cloud providers.
+
+<highlight type="note">
+If you are running a Managed Service for TimescaleDB 
+[Pro plan](https://docs.timescale.com/mst/latest/about-mst/#service-configuration-plans),
+you have standby nodes available in a high availability setup. The standby nodes support
+read-only queries to reduce the effect of slow queries on the primary node.
+</highlight>
+
```

Comment:
can we add a note that the service must be in `Running` before starting the process?

Reply:
sure!!! thank you @jfjoly !!! the suggestions you made are coming in a new PR because this is merged now :(